compose/config/interpolation.py

```diff
@@ -8,13 +8,22 @@
 log = logging.getLogger(__name__)
 
 
-def interpolate_environment_variables(config):
+def interpolate_environment_variables(config, version):
     mapping = BlankDefaultDict(os.environ)
+    cfg = config.copy()
```

Comment:
I'm generally against abbreviated variable names because they are harder to read, in python we value verbosity because it gives readability to the code and clearer intent. Please use descriptive variable names.